cypress/e2e/testAddAdminDescription.cy.js

```diff
@@ -0,0 +1,27 @@
+/// <reference types='cypress' />
+
+let people;
+
+describe("Add user description", ()=>{
+
+    beforeEach(function (){
```

Comment:
beforeEach this needs to be removed. Because this code was added to hooks and it works for all tests